src/modules/auth/api.js

```diff
@@ -86,9 +86,13 @@ export class SeaCatAuthApi {
 	}
 
 	// Verify access to tenant
-	verify_access(tenant, access_token, resource) {
+	verify_access(access_token, tenant, resource) {
```

Comment:
The comment is incorrect, because it is assuming that the `tenant` is present.
It is verifying access to the application, using access_token and optionally also tenant.

`tenant` argument should be last, because it could be undefined.

Reply:
Refactored in those 2 commits: 
https://github.com/TeskaLabs/asab-webui/pull/105/commits/09439c4d06d328afff11a3498d4750585b02b18d & https://github.com/TeskaLabs/asab-webui/pull/105/commits/72c1e71b665b62dc179ae6d360b04843e6cf2db1